core/src/mindustry/ui/fragments/ChatFragment.java

```diff
@@ -198,7 +198,11 @@ private void sendMessage(){
         //avoid sending prefix-empty messages
         if(message.isEmpty() || (message.startsWith(mode.prefix) && message.substring(mode.prefix.length()).isEmpty())) return;
 
-        history.insert(1, message);
+        if(!mode.prefix.isEmpty() && message.length() > mode.normalizedPrefix().length()) {
+            history.insert(1, message.substring(mode.normalizedPrefix().length()));
+        } else {
```

Comment:
Does not fit formatting guidelines, see CONTRIBUTING.md.

Reply:
Sorry. Now I formatted it, and just in case I also compared each item in CONTRIBUTING.md with the code, I hope everything is fine now